markdown/docs/reference/specification/v3.0.0-explorer.md

```diff
@@ -0,0 +1 @@
+<Visualizer />
```

Comment:
@AceTheCreator  Can you please add metadata for this page, as it will be used as title and description in the head tag?